### ISSUE TITLE
Fat Item + Sprite option?

### ISSUE THREAD

**Issue by WMan22** [hh79/gzdoomvr#2]
Is there any way that this can get updated with a fat item + sprite option? I like the physicality of having fat item, but I like the detail of having a sprite. The crossed thing looks weird with its ^ formation.

**Comment by hh79** [comment]
Yeah it would be nice, unfortunately there are no higher resolution weapon sprites except for hud. The fat item is rendered with a low resolution spawn sprites.  When you look closer at a weapon pickup, you will see the same ugly pixels.

**Comment by WMan22** [comment]
Nonono, what I mean is like, render fat item VR option for weapons, but put the sprite behind it. Crossed is a problem because it puts the sprites in a triangle and not horizontal. I don't mind that the fat item is low resolution, 

I just want a combo of "sprite only" and "fat item" in one package, because crossed does this weird thing where the world sprite is triangular and that makes it harder to aim.

**Comment by WMan22** [comment]
> Yeah it would be nice, unfortunately there are no higher resolution weapon sprites except for hud. The fat item is rendered with a low resolution spawn sprites. When you look closer at a weapon pickup, you will see the same ugly pixels.

Nonono, what I mean is like, render fat item VR option for weapons, but put the sprite behind it. Crossed is a problem because it puts the sprites in a triangle and not perfectly vertical. I don't mind that the fat item is low resolution,

I just want a combo of "sprite only" and "fat item" in one package, because crossed does this weird thing where the world sprite is triangular and that makes it harder to aim. I want another option to be added that is basically fat item + sprite only, that way you get the proper angle of fat item but the detail of a sprite, because the existing "crossed" option isn't cutting it for that.

**Comment by hh79** [comment]
Ok, now I got it, definitely it is an interesting idea.
